components/camel-etcd/pom.xml

```diff
@@ -109,17 +109,47 @@
                 </plugins>
             </build>
         </profile>
+
+        <!-- activate test if the docker socket file is accessible -->
+        <profile>
+            <id>etcd-tests-docker-file</id>
+            <activation>
+                <file>
+                    <exists>/var/run/docker.sock</exists>
```

Comment:
Consider also checking Windows and npipe

Reply:
Do you have an example. I don't have any windows machine around :)